src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task {
+
+    protected String at;
```

Comment:
You might want to reconsider the variable name "at" also, it's unclear about the meaning

Reply:
I will consider that.